packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -241,24 +242,33 @@ impl Modify for ExternalRefAddon {
     }
 }
 
-fn modify_component_references(content: &mut HashMap<String, openapi::Content>) {
+fn modify_component_body(content: &mut BTreeMap<String, openapi::Content>) {
     for content in content.values_mut() {
         modify_schema_references(&mut content.schema);
     }
 }
 
-fn modify_schema_references(schema_component: &mut openapi::Component) {
+// TODO: can we avoid importing IndexMap here?
```

Comment:
There doesn't seem to be a trait we can rely on? Maybe we can pass a mutable iterator in here instead of taking the map?

Reply:
(I was hoping all Maps would inherit from some `Map` trait but it turns out there are good reasons for not doing so)

Reply:
Removed it with `IntoIterator`